src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -213,14 +218,65 @@ public long GetHistogramCount()
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public double GetHistogramSum()
         {
-            if (this.aggType != AggregationType.Histogram && this.aggType != AggregationType.HistogramSumCount)
+            if (this.aggType != AggregationType.Histogram &&
+                this.aggType != AggregationType.HistogramMinMax &&
+                this.aggType != AggregationType.HistogramSumCount &&
+                this.aggType != AggregationType.HistogramSumCountMinMax)
             {
                 this.ThrowNotSupportedMetricTypeException(nameof(this.GetHistogramSum));
             }
 
             return this.histogramBuckets.SnapshotSum;
         }
 
+        /// <summary>
+        /// Gets the minimum value of the histogram associated with the metric point.
+        /// </summary>
+        /// <remarks>
+        /// Applies to <see cref="MetricType.Histogram"/> metric type.
+        /// </remarks>
+        /// <returns>Minimum value.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public double GetHistogramMin()
+        {
+            if (this.aggType != AggregationType.HistogramMinMax &&
+                this.aggType != AggregationType.HistogramSumCountMinMax)
+            {
+                this.ThrowNotSupportedMetricTypeException(nameof(this.GetHistogramMin));
+            }
+
+            if (this.runningValue.AsLong == 0)
+            {
+                throw new InvalidOperationException("No values have been recorded yet.");
+            }
+
+            return this.histogramBuckets.SnapshotMin;
+        }
+
+        /// <summary>
+        /// Gets the maximum value of the histogram associated with the metric point.
+        /// </summary>
+        /// <remarks>
+        /// Applies to <see cref="MetricType.Histogram"/> metric type.
+        /// </remarks>
+        /// <returns>Maximum value.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public double GetHistogramMax()
+        {
+            if (this.aggType != AggregationType.HistogramMinMax &&
+                this.aggType != AggregationType.HistogramSumCountMinMax)
+            {
+                this.ThrowNotSupportedMetricTypeException(nameof(this.GetHistogramMax));
+            }
+
+            if (this.runningValue.AsLong == 0)
```

Comment:
would this condition ever occur?

Reply:
Here's the test case for it https://github.com/open-telemetry/opentelemetry-dotnet/pull/2735/files#diff-a3e99c5f40d4f5df6f97f7c944d7c4b560a70b57fc299321c5b656757cf0d76cR651

Are you asking that when we export and there are no items then we do nothing so having this check is redundant? In that case I think so, I could replace with a debug.assert, since the exporters haven't been modified to include the min/max values yet.